praw/models/reddit/subreddit.py

```diff
@@ -882,6 +882,12 @@ def edited(self, only=None, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To print all items in the edited queue try:
+        .. code:: python
+
+          for item in reddit.subreddit('mod').mod.edited(limit=None):
```

Comment:
Good to know. Thanks for the info. Looks like that part should probably be updated to have 4-space indents.